cnc_ctrl_v1/Kinematics.cpp

```diff
@@ -235,14 +241,18 @@ void  Kinematics::forward(const float& chainALength, const float& chainBLength,
         
         guessCount++;
 
-        //Prevent the connection from timing out
```

Comment:
Understood. I left the output in as a debugging option, I think it could still be helpful.  It will now just output it's position and error data on the defined schedule if this takes too long.